testing/openresty/APKBUILD

```diff
@@ -0,0 +1,71 @@
+# Contributor: Yichun Zhang <agentzh@gmail.com>
+# Maintainer: Yichun Zhang <agentzh@gmail.com>
+pkgname=openresty
+pkgver=1.13.6.1
+pkgrel=0
+pkgdesc="Scalable Web Platform by Extending NGINX with Lua"
+url="https://openresty.org/"
+arch="all"
+license="BSD"
```

Comment:
The license should be in the format provided by SPDX. In this case, is 'BSD-3-Clause`.